charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -26,6 +26,42 @@
     "Attribute": {
```

Comment:
Please rename to InfraAttributeName.  However, do we really need this definition?  Can't there just be a string in InfraAttributes?

Reply:
Done